view/frontend/web/js/components/steps/BaseShippingStep.vue

```diff
@@ -106,16 +207,31 @@ export default {
   },
   data() {
     return {
-      baseUrl            : baseUrl,
-      config             : this.$store.state.config,
       isRegionIdHidden   : false,
       regionList         : regionList,
       shippingAddress    : shippingAddress,
       shippingInformation: this.$store.state.shippingInformation,
-      totals             : this.$store.state.totals
+      totals             : this.$store.state.totals,
+      email: '',
+      firstname: '',
+      lastname: '',
+      telephone: '',
+      streetFirstRow: '',
+      streetSecondRow: '',
+      countryId: '',
+      city: '',
+      postcode: '',
+      regionId: '',
+      company: '',
+      countries,
+      shippingMethod: '',
+      regions: []
     };
   },
   computed: {
+    baseUrl() {
```

Comment:
Why you are adding here computed with baseUrl when we don't use it anywhere? Do you have a plan?